corehq/apps/app_manager/templates/app_manager/partials/mobile_report_configs.html

```diff
@@ -9,6 +9,12 @@
                 <input type="text" data-bind="value: currentModuleName">
             </div>
         </div>
+        <div class="control-group">
```

Comment:
does anything need to be restricted in the ui based on the toggle? it looks like the old template used to: app_manager/partials/module_filter.html